core/domain/user_services.py

```diff
@@ -241,6 +265,78 @@ def validate(self):
                 '%s is not a valid value for the dashboard display '
                 'preferences.' % (self.creator_dashboard_display_pref))
 
+    def populate_from_user_data(self, modifiable_user_data):
+        """Populate the UserSettings domain object using the user data in
+            modifiable_user_data.
+
+        Args:
+            modifiable_user_data: ModifiableUserData. The modifiable user
+                data version_1 object with the data information to be updated.
```

Comment:
Typically this should be the latest schema version?

If so, you can just drop "version_1" -- latest schema version is assumed.

Reply:
Yes it should be the latest version only. I have modified the string now.